src/components/LocationPage/Notifications.tsx

```diff
@@ -50,37 +47,27 @@ const NotificationArea: React.FC<{ projections: Projections }> = ({
   const region = projections.region;
 
   enum Notification {
-    NYCCounty,
     HospitalizationsPeak,
-    ThirdWave,
     ExposureNotifications,
-    TXFeb2021Winter,
+    Vulnerability,
     None,
   }
   let notification: Notification;
 
-  if (showExposureNotifications(region)) {
+  // NOTE: For now showVulnerability() always returns true and will be the notification shown.
+  if (showVulnerability(region)) {
+    notification = Notification.Vulnerability;
+  } else if (showExposureNotifications(region)) {
     notification = Notification.ExposureNotifications;
-  } else if (
-    // TODO(2021/2/20): TX Winter Reporting
-    // https://trello.com/c/TdspuIeM/952-texas-reporting-dip-winter-weather-feb-2020
-    getFormattedStateCode(region)?.includes('TX')
-  ) {
-    notification = Notification.TXFeb2021Winter;
-  } else if (
-    // TODO(2020/12/22): Remove NYC notice after it's been up for a week or so.
-    ['36047', '36061', '36005', '36081', '36085'].includes(region.fipsCode)
-  ) {
-    notification = Notification.NYCCounty;
   } else if (isHospitalizationsPeak(projections.primary)) {
```

Comment:
If we are cleaning house should we remove this too.

By the time this could trigger again, we would be solidly into a 4th wave and likely already want different messaging. And if we need this type of logic we could also revert the changes later.